package Protos;

import "base.proto";

message TimeSeriesInfo {
    repeated TStringToUintPair unique_labels = 1;
    repeated TUintToUintPair labels_start = 2;
    required double dt = 3;
}

